macros/edr/dbt_artifacts/upload_dbt_artifacts.sql

```diff
@@ -8,7 +8,6 @@
       "dbt_metrics": elementary.upload_dbt_metrics,
       "dbt_exposures": elementary.upload_dbt_exposures,
       "dbt_seeds": elementary.upload_dbt_seeds,
-      "dbt_columns": elementary.upload_dbt_columns,
```

Comment:
Hmm I think it's problematic to remove it from here - it means `dbt_columns` won't be cached.
Can you add the `disable_dbt_columns_autoupload` logic here instead?